dev/com.ibm.ws.jca_fat/fat/src/com/ibm/ws/jca/fat/app/JCATest.java

```diff
@@ -134,27 +101,28 @@ public static void tearDown() throws Exception {
 
     @Test
     public void testActivationSpec() throws Exception {
-        runInServlet("testActivationSpec", fvtweb);
+        runTest();
     }
 
     @Test
     public void testActivationSpecBindings() throws Exception {
-        runInServlet("testActivationSpecBindings", fvtweb);
+        runTest();
     }
 
     @Test
     public void testDestinations() throws Exception {
-        runInServlet("testDestinations", fvtweb);
+        runTest();
     }
 
     @Test
+    @SkipForRepeat(EE9_FEATURES) //Transformer does not seem to transform jars inside of jars
```

Comment:
It's fine to skip the test, but this seems like something to report to the team that is working on the transformer if you haven't already.

Reply:
👍 Will do!